README.adoc

```diff
@@ -316,13 +326,16 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see whether the tests detect a failure, change the base URL in the configuration file so that when the `inventory` service tries to access the invalid URL, an UnknownUrlException is thrown.
+To see whether the tests detect a failure, change the base URL in the configuration file so that when the `inventory` service tries to access the invalid URL, an UnknownUriException is thrown.
 Rerun the Maven build. You see a test failure occur.
 
 == Great work! You're done!
 
 You just invoked a remote service by using a template interface with MicroProfile Rest Client in Open Liberty.
 
+MicroProfile Rest Client also provides a uniform way to configure SSL for the client.
+You can learn more in the https://openliberty.io/blog/2019/06/21/microprofile-rest-client-19006.html#ssl[Hostname verification with SSL on Open Liberty and MicroProfile Rest Client^] blog and the https://github.com/eclipse/microprofile-rest-client/releases[MicroProfile Rest Client specification^].
+
 Feel free to try one of the related guides. They demonstrate more technologies that you can learn and
```

Comment:
change
Feel free to try one of the related guides. They demonstrate more technologies that you can learn and
expand on what you built here.
TO
**Try one of the other related guides where you can learn more technologies and expand on what you built here.**

Reply:
Updated